sql/src/main/java/io/crate/operation/reference/sys/check/checks/AbstractSysCheck.java

```diff
@@ -25,13 +25,20 @@
 
 public abstract class AbstractSysCheck implements SysCheck {
 
+    private static final String LINK_PATTERN = "https://cr8.is/d-check-";
+
     private final int id;
     private final BytesRef description;
     private final Severity severity;
 
+
     public AbstractSysCheck(int id, BytesRef description, Severity severity) {
```

Comment:
I think by changing the description in the arguments to `String` it would be possible to avoid one BytesRef -> utf8String roundtrip since the sub-classes have a string anway